logicalplan/plan.go

```diff
@@ -10,6 +10,10 @@ import (
 	"github.com/prometheus/prometheus/promql/parser"
 )
 
+var NoOptimizers = []Optimizer{}
+
+var Optimizers = append(DefaultOptimizers, PropagateMatchersOptimizer{})
```

Comment:
What's this? Can we add commentary?

Reply:
I renamed it to `AllOptimizers` to make it more clear.